modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -0,0 +1,164 @@
+====
+
+Perform the following procedure to subscribe to bare-metal events for the node using a `BMCEventSubscription` CR.
+
+.Prerequisites
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Get the user name and password for the BMC.
+* Deploy a bare-metal node with a Redfish-enabled Baseboard Management Controller (BMC) in your cluster, and enable Redfish events on the BMC.
++
+[NOTE]
+====
+Enabling Redfish events on specific hardware is outside the scope of this information. For more information about enabling Redfish events for your specific hardware, consult the BMC manufacturer documentation.
+====
+
+.Procedure
+. Confirm that the node hardware has the Redfish `EventService` enabled by running the following `curl` command:
++
+[source,terminal]
+----
+curl https://<bmc_ip_address>/redfish/v1/EventService --insecure -H 'Content-Type: application/json' -u "<bmc_username>:<password>"
+----
++
+where:
++
+--
+bmc_ip_address:: is the IP address of the BMC where the Redfish events are generated.
+--
++
+.Example output
+[source,terminal]
+----
+{
+   "@odata.context": "/redfish/v1/$metadata#EventService.EventService",
+   "@odata.id": "/redfish/v1/EventService",
+   "@odata.type": "#EventService.v1_0_2.EventService",
+   "Actions": {
+      "#EventService.SubmitTestEvent": {
+         "EventType@Redfish.AllowableValues": ["StatusChange", "ResourceUpdated", "ResourceAdded", "ResourceRemoved", "Alert"],
+         "target": "/redfish/v1/EventService/Actions/EventService.SubmitTestEvent"
+      }
+   },
+   "DeliveryRetryAttempts": 3,
+   "DeliveryRetryIntervalSeconds": 30,
+   "Description": "Event Service represents the properties for the service",
+   "EventTypesForSubscription": ["StatusChange", "ResourceUpdated", "ResourceAdded", "ResourceRemoved", "Alert"],
+   "EventTypesForSubscription@odata.count": 5,
+   "Id": "EventService",
+   "Name": "Event Service",
+   "ServiceEnabled": true,
+   "Status": {
+      "Health": "OK",
+      "HealthRollup": "OK",
+      "State": "Enabled"
+   },
+   "Subscriptions": {
+      "@odata.id": "/redfish/v1/EventService/Subscriptions"
+   }
+}
+----
+
+. Get the {redfish-operator} service route for the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get route -n openshift-bare-metal-events
+----
++
+.Example output
+[source,terminal]
+----
+NAME             HOST/PORT                                                       PATH   SERVICES                 PORT   TERMINATION   WILDCARD
+hw-event-proxy   hw-event-proxy-cloud-native-events.apps.compute-1.example.com          hw-event-proxy-service   9087   edge          None
```

Comment:
@jzding  could you help verify this output? Thanks.

Reply:
The output of 
```
$ oc get route -n openshift-bare-metal-events
```
should be the following, where `hw-event-proxy` is the service name and `openshift-bare-metal-events` is namespace.
```
NAME             HOST/PORT                                                                                           PATH   SERVICES                 PORT   TERMINATION   WILDCARD
hw-event-proxy   hw-event-proxy-openshift-bare-metal-events.apps.compute-1.example.com          hw-event-proxy-service   9087   edge          None
```